toolbar/toolbar-feature-flags-sveltekit/src/routes/+page.server.ts

```diff
@@ -0,0 +1,8 @@
+import { shouldUseSvelteColor } from '$lib/flags'
+
+export async function load() {
+  return {
+    // Requests the feature flag and returns its value
+    useSvelteColor: await shouldUseSvelteColor(),
```

Comment:
Will people think about hooks when we prefix this with "use"?

Reply:
React-heads might yeah - in fact eslint did error when I called the flag `useSvelteColor` 😆 I'll come up with a different name